Description: Comment out some calls that need to be performed at install time
Author: Brian Thomason <brian.thomason@canonical.com>
--- rvm-1.6.9.orig/scripts/install
+++ rvm-1.6.9/scripts/install
@@ -1,17 +1,26 @@
 #!/usr/bin/env bash
 
-rvm_load_rvm=0
+shopt -s extglob
+PS4="+ \${BASH_SOURCE##\${rvm_path:-}} : \${FUNCNAME[0]:+\${FUNCNAME[0]}()}  \${LINENO} > "
+export PS4
+set -o errtrace
+
+# Tracing, if asked for.
+if [[ "$*" =~ --trace ]] || (( ${rvm_trace_flag:-0} > 0 ))
+then
+  set -o xtrace
+  export rvm_trace_flag=1
+fi
 
-source "scripts/rvm"
-source "scripts/functions/installer"
+source "$PWD/scripts/functions/installer"
+# source "$PWD/scripts/rvm"
 
 #
 # RVM Installer
 #
-determine_install_path
-determine_install_or_upgrade
 install_setup
 
+true ${DESTDIR:=}
 # Parse RVM Installer CLI arguments.
 while (( $# > 0 ))
 do
@@ -58,6 +67,22 @@ do
       ;;
   esac
 done
+
+if [[ -n "${DESTDIR}" ]]
+then
+  rvm_prefix="${DESTDIR}"
+fi
+
+determine_install_path
+
+determine_install_or_upgrade
+
+if [[ -z "${rvm_path:-}" ]]
+then
+  echo "ERROR: rvm_path is empty, halting installation."
+  exit 1
+fi
+
 export rvm_prefix rvm_path rvm_debug_flag rvm_trace_flag
 
 create_install_paths
@@ -66,7 +91,7 @@ print_install_header
 
 configure_installation
 
-cleanse_old_entities
+#cleanse_old_entities
 
 install_rvm_files
 
@@ -76,32 +101,32 @@ setup_configuration_files
 
 install_binscripts
 
-automatic_profile_setup
+#automatic_profile_setup
 
 install_gemsets
 
 install_patchsets
 
-cleanse_old_environments
+#cleanse_old_environments
 
-migrate_old_gemsets
+#migrate_old_gemsets
 
-migrate_defaults
+#migrate_defaults
 
 correct_binary_permissions
 
-install_man_pages
+#install_man_pages
 
-root_canal
+#root_canal
 
-setup_rvmrc
+#setup_rvmrc
 
-setup_user_profile
+#setup_user_profile
 
 cleanup_tmp_files
 
-display_notes
+#display_notes
 
-display_thank_you
+#display_thank_you
 
 exit 0
